examples/jsm/loaders/LDrawLoader.js

```diff
@@ -767,7 +767,7 @@ class LDrawParsedCache {
 				const text = await fileLoader.loadAsync( subobjectURL );
 				return text;
 
-			} catch {
+			} catch ( _ ) {
```

Comment:
Maybe we can add also Optional Catch Binding to the linter? It has appeared in the specification only in ES2019 but was supported everywhere for a long time.

Reply:
Could you maybe do that in a follow-up PR? I'm unfortunately not familiar with configuring the linter (yet)

Reply:
I'm not very familiar with it too, actually 😅